volumina/view3d/glview.py

```diff
@@ -39,6 +47,31 @@ def __init__(self, parent=None):
         size_policy.setRetainSizeWhenHidden(True)
         self.setSizePolicy(size_policy)
 
+    def gl_version_sufficient(self) -> Tuple[bool, str]:
```

Comment:
It seems like [`glGetString(GL_VERSION)`](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glGetString.xhtml) can also a return version string in the `MAJOR.MINOR.RELEASE` form.
Also, comparing floats works in this case, but feels somewhat hacky.

Maybe do it in the following way?

```python
_MININUM_GL_VERSION = (2, 0, 0)

def gl_version(self):
    version = tuple(map(int, glGetString(GL_VERSION).decode().partition(" ")[0].split(".")))
    assert len(version) in (2, 3)
    if len(version) == 2:
        version = version + (0,)
    return version
```

Reply:
True that, but I didn't find any guarantees in the openGL docs. But I'll add something in those terms.